src/main/java/com/bergerkiller/bukkit/tc/TCConfig.java

```diff
@@ -341,7 +342,18 @@ public static void load(FileConfiguration config) {
         config.addHeader("activatorEjectEnabled", "If activator rails are used for decoration purposes, this should be disabled");
         activatorEjectEnabled = config.get("activatorEjectEnabled", true);
 
+        config.setHeader("enabledWorlds", "\nA list of world names where TrainCarts should be enabled");
+        config.addHeader("enabledWorlds", "World names are not case-sensitive");
+        enabledWorlds.clear();
+        if (!config.contains("enabledWorlds")) {
+            config.set("enabledWorlds", Arrays.asList("DefaultWorld1", "DefaultWorld2"));
```

Comment:
Dont set to DefaultWorld1/DefaultWorld2, set to Collections.emptyList() instead.